setup.py

```diff
@@ -859,7 +859,11 @@ def debug_build():
         license="HPND",
         author="Alex Clark (PIL Fork Author)",
         author_email="aclark@python-pillow.org",
-        url="http://python-pillow.org",
+        url="https://python-pillow.org/",
+        project_urls={
+            "Documentation": "https://pillow.readthedocs.io",
```

Comment:
```suggestion
            "Documentation": "https://pillow.readthedocs.io",
            "Funding": "https://tidelift.com/subscription/pkg/pypi-pillow",
```
Like Django, we could also add in a funding link.